### PR TITLE
Reorganize project

### PR DESCRIPTION
**Pull request by j8r**
Move files to the top level, and use more features of the DPPM library like the CLI

### CODE REVIEW

File: src/cli.cr [hunk 1]
```diff
@@ -11,14 +11,20 @@ module DppmRestApi::CLI
       port ||= dppm_config.port
       host ||= dppm_config.host
     end
-    DppmRestApi.run port, host
+    DppmRestApi.run host, port, data_dir
```
Comment (dscottboggs) [inline src/cli.cr:14]: The other options are already set by DPPM?

File: src/route.cr [hunk 2]
```diff
@@ -0,0 +1,28 @@
+require "./access"
```
Comment (dscottboggs) [inline src/route.cr]: I think this file should be namespaced under `DppmRestApi::Config`, and be at src/config/route.cr.

**Review by dscottboggs — CHANGES_REQUESTED** [review]
Thanks for bringing the inconsistency to my attention :+1:

File: src/cli.cr (hunk 1 above)
Comment (j8r) [reply src/cli.cr:14]: Yes, this variables comes from the CLI, which has default values (below).

**Comment by j8r** [comment]
Changes done, as you suggested (I killed the first CI job, the other one pass).

**Review by dscottboggs — APPROVED** [review]

**Pull request merged**